.github/workflows/test.yaml

```diff
@@ -31,12 +31,3 @@ jobs:
       - uses: actions/checkout@v2
       - run: yarn install --frozen-lockfile --ignore-optional
       - run: ./scripts/smoke.sh
-
-  coverage:
-    needs: [test]
-    runs-on: ubuntu-latest
-    steps:
-      - uses: coverallsapp/github-action@master
```

Comment:
The GitHub org doesn't let us use external actions outside our space. Dropping coveralls reports (nice to have, and not fundamental).